### PR TITLE
Ruby 2.7 and 2.6.5

### LINKED ISSUE #1683: Compatibility with Ruby 2.7?
@feliperenan  👍

### PR DESCRIPTION
**Pull request by pedrofurtado**
Closes #1683 
@feliperenan @tegon

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ rvm:
   - 2.4.6
   - 2.5.5
   - 2.6.3
+  - 2.6.5
```
Comment (tegon) [inline .travis.yml]: Can you replace version `2.6.3` with `2.6.5` instead? We try to keep only one build per minor version.

Comment (pedrofurtado) [reply .travis.yml]: @tegon thanks for feedback! 🤝 i will make this change

Comment (pedrofurtado) [reply .travis.yml]: @tegon done 👍 🎉

**Comment by pedrofurtado** [comment]
@feliperenan @tegon Reading the output of travis, the ruby 2.7 and 2.6.5 is compatible with gem 🥇

**Comment by feliperenan** [comment]
Hi @pedrofurtado :wave: 

Could you rebase your branch? we pushed a CI fix to the master branch that probably will fix this PR build as well.

Thanks for your contribution :muscle:

**Comment by pedrofurtado** [comment]
@feliperenan Rebased 🏗 🤝

**Comment by pedrofurtado** [comment]
@feliperenan Rebased 🏗 🤝

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+### Enhancements
+* Compatibility with ruby 2.7.0 and 2.6.5. [@tegon](https://github.com/tegon), [@pedrofurtado](https://github.com/pedrofurtado) and [@feliperenan](https://github.com/feliperenan)
```
Comment (rafaelfranca) [inline CHANGELOG.md]: This don't need a changelog entry. It has no code change, so it means that the current version is already compatible.

Comment (pedrofurtado) [reply CHANGELOG.md]: @rafaelfranca Thanks for feedback! Done 🤝

**Pull request merged**